packages/commonwealth/client/scripts/controllers/app/login.ts

```diff
@@ -6,6 +6,7 @@ import { WalletId } from 'common-common/src/types';
 import { notifyError } from 'controllers/app/notifications';
 import { isSameAccount } from 'helpers';
 import axios from 'axios';
+import $ from 'jquery';
```

Comment:
use axios instead of jquery

Reply:
Maybe out of scope for now. This is just a fix to fix the login on experimental